library/swift/src/Envoy.swift

```diff
@@ -47,20 +47,9 @@ public final class Envoy: NSObject {
 }
 
 extension Envoy: Client {
-  public func startStream(with request: Request, handler: ResponseHandler) -> StreamEmitter {
+  public func send(_ request: Request, handler: ResponseHandler) -> StreamEmitter {
```

Comment:
Would `send(request:` be clearer than `send(_ request:`? Genuine question - I don't actually know which is more idiomatic Swift.

Reply:
I think this is common practice since type inference of `Request` is available from the compiler